app/views/assigned_applications/index.html.erb

```diff
@@ -6,44 +6,52 @@
     <p><%=t '.your_assignments', count: @applications.count %></p>
   </div>
 </div>
+
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full govuk-!-margin-bottom-5">
-    <%= button_to(
-          'Get next application',
-          get_next_assigned_applications_path,
-          method: :post,
-          class: 'govuk-button'
-        ) %>
+    <%= form_with url: get_next_assigned_applications_path, method: :post do |f| %>
+      <div class="govuk-button-group">
+        <%= f.button t('.get_next'),
+                     class: 'govuk-button',
+                     data: { module: 'govuk-button' } %>
+
+        <%= link_to t('.open_applications'),
+                    crime_applications_path,
+                    class: 'govuk-link--no-visited-state govuk-!-font-size-19' %>
+      </div>
+    <% end %>
   </div>
 </div>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
-    <table class="govuk-table app-dashboard-table">
-      <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th scope="col" class="govuk-table__header">
-            <%= t('.table_headings.applicant_name') %>
-          </th>
-          <th scope="col" class="govuk-table__header">
-            <%= t('.table_headings.reference') %>
-          </th>
-          <th scope="col" class="govuk-table__header">
-            <%= t('.table_headings.received_at') %>
-          </th>
-          <th scope="col" class="govuk-table__header">
-            <%= t('.table_headings.time_passed') %>
-          </th>
-          <th scope="col" class="govuk-table__header">
-            <%= t(:common_platform, scope: 'crime_applications.labels') %>
-          </th>
-        </tr>
-      </thead>
-      <tbody class="govuk-table__body">
-        <% @applications.each do |app| %>
-          <%= render 'application/crime_application', crime_application: app %>
-        <% end %>
-      </tbody>
-    </table>
+<% if @applications.any? %>
+  <div class="govuk-grid-row">
```

Comment:
Wondering if we could wrap this table in a partial - might make this view a bit easier to parse.

Reply:
Done